### DISCUSSION TITLE
Mermaid chart integration

### DISCUSSION THREAD

**Discussion by truongdo** [outline/outline#2870] [Ideas]
It would be great to have mermaid integrated with outline. It's a great chart and diagram visualization using markdown definitions.

https://github.com/mermaid-js/mermaid

**Comment by BRogers80** [comment]
Yes,please. +1

**Comment by kengjungwa** [comment]
+1 too. 
It'd be very convenient and seamless to be able to edit mermaid diagrams within Outline as we do our design documentation.

**Comment by Shide** [comment]
PR: https://github.com/outline/outline/pull/3523
Transferred to https://github.com/outline/outline/pull/3679

**Comment by tommoor** [comment]
As part of this work I just reduced the size of mermaidjs for everyone by 8% 😆 

https://github.com/mermaid-js/mermaid/pull/3167
